common/lib/xmodule/xmodule/public/js/vertical_student_view.js

```diff
@@ -0,0 +1,16 @@
+/* JavaScript for Vertical Student View. */
+window.VerticalStudentView = function (runtime, element) {
+
+    RequireJS.require(['js/bookmarks/views/bookmark_button'], function (BookmarkButton) {
```

Comment:
@dan-f That's a really good question. I'm a little uncomfortable with using RequireJS like this, because I don't really understand all of the ramifications. The video xblock does some strange things where it uses RequireJS to define dependencies, but then it also uses a numbering scheme to ensure that the xmodule bundled file will have the definitions in the correct order. This xmodule is probably okay, because the dependency is to a file that isn't in the xmodule bundle, but it should be verified carefully on a sandbox. @muzaffaryousaf, is my explanation correct here? We probably should update the wiki page to mention this case too.